service/src/main/resources/config/aou/test/SC2023Q3R1/sql/t_visit.sql

```diff
@@ -0,0 +1,11 @@
+SELECT
+  distinct c.concept_id, c.concept_name
+
+FROM `all-of-us-ehr-dev.SC2023Q3R1.visit_occurrence` AS vo
```

Comment:
Ditto comment for device sql above.

Reply:
Same, if a particular visit is not observed (no-person has such a visit) we do not show that visit ref:[build-cb-criteria-visit.sh](https://github.com/all-of-us/workbench/blob/a909b2097bfa407c050fdaa5b2b351a212f4f8c1/api/db-cdr/generate-cdr/build-cb-criteria-visit.sh#L57-L63). Any row where concept_id counts is 0 are removed see [cb_criteria-clean-up](https://github.com/all-of-us/workbench/blob/a909b2097bfa407c050fdaa5b2b351a212f4f8c1/api/db-cdr/generate-cdr/build-cb-criteria-attribute-tables-and-cleanup.sh#L1-L1)